guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
+| `sdk-version`                   | No                                                  | 0.0.0          | string  | semantic version           | Version of the SDK used to generate the manifest.                                                          |
+| `timeout`                       | No                                                  | 0              | integer | number                     | The maximum number of seconds the application should execute before being timed out and terminated.        |
+| `version`                       | No                                                  | 0.0.0          | string  | semantic version           | Version of the Application.                                                                                |
+| `working-directory`             | No                                                  | /var/holoscan/ | string  | absolute file-system path  | Folder, or directory, in which the application will be executed.                                           |
+
+### Package Manifest
+
+The Package Manifest file provides information about the MAP's package layout. It is not intended as a mechanism for controlling how the MAP is used or how the MAP's Application is executed.
 
 - Package Manifest MUST be UTF-8 encoded and use the JavaScript Object Notation (JSON) format.
 
-- Package Manifest SHOULD support either CRLF and LF style line endings.
+- Package Manifest SHOULD support either CRLF or LF style line endings.
 
-- Package Manifest SHOULD specify the folder which contains the Application (`/etc/monai/pkg.json#application-root`).
+- Package Manifest SHOULD specify the folder which contains the application (`/etc/holoscan/pkg.json#application-root`).
 
-  - When not provided the default path `/opt/monai/app/` will be assumed.
+  - When not provided, the default path `/opt/holoscan/app/` will be assumed.
 
-- Package Manifest SHOULD provide the version of the package file manifest schema (`/etc/monai/pkg.json#api-version`).
+- Package Manifest SHOULD provide the version of the package file manifest schema (`/etc/holoscan/pkg.json#api-version`).
 
   - Manifest schema version SHALL be provided as a [semantic version](https://semver.org/) string.
 
-- Package Manifest SHOULD provide the version of the tools used to build the package (`/etc/monai/pkg.json#sdk-version`).
+- Package Manifest SHOULD provide the version of the tools used to build the package (`/etc/holoscan/pkg.json#sdk-version`).
 
   - SDK version SHALL be provided as a [semantic version](https://semver.org/) string.
 
-- Package Manifest SHOULD provide the version of the package itself (`/etc/monai/pkg.json#version`).
+- Package Manifest SHOULD provide the package version of itself (`/etc/holoscan/pkg.json#version`).
```

Comment:
There is an explosion of version numbers. Understandably this is different from the application version number, but how is this one related to the version number in the container annotations?